dependencies.yml

```diff
@@ -161,8 +161,8 @@ org.junit.jupiter:
 kr.motd.gradle:
   sphinx-gradle-plugin: { version: '2.10.1' }
 
-me.champeau.jmh:
-  jmh-gradle-plugin: { version: '0.6.4' }
+me.champeau.gradle:
+  jmh-gradle-plugin: { version: '0.5.3' }
```

Comment:
How about rollbacking to `0.5.0`?
https://github.com/line/armeria/pull/3561/files#diff-1d8ca348d900d958b9aa55ebd96110af6deb206336d2f7c9778b7d8bcd60f99bL334

Reply:
https://github.com/line/centraldogma/pull/598/files#diff-1d8ca348d900d958b9aa55ebd96110af6deb206336d2f7c9778b7d8bcd60f99bL164
It was 0.5.3 and it works well so let me just use the version. 😉

Reply:
OK, go ahead.